Source/SchemaGenerator.cs

```diff
@@ -54,6 +54,7 @@ public IEnumerable<ExplorerItem> GetSchema()
 				.Where(p =>
 					p.GetCustomAttributeLike<ObsoleteAttribute>() == null &&
 					p.PropertyType.MaybeChildOf(typeof(IQueryable<>)))
+				.OrderBy(p => p.Name)

```

Comment:
I suggest to sort by [`ColumnAttribute.Order`](https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/Mapping/ColumnAttribute.cs#L330-L340) for columns with `HasOrder` and by name for rest

Reply:
The original change will sort the tables.  I'm still trying to sort the columns but I will honor the ColumnAttribute if the order is specified.

Reply:
not sure of helpful, but here we apply similar sorting for columns in create table based on Order: https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/SqlProvider/BasicSqlBuilder.cs#L1111

Reply:
Tables and columns shold now sort by name honouring column attribute order if applicable